### PR TITLE
Small typo in doc, code doesnt run without .+

### PR DESCRIPTION
**Pull request by eirikbrandsaas**
I was having some problems so went back to basics, but the code didn't run!

### CODE REVIEW

File: docs/src/user/tipsandtricks.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ on three inputs, and two of them are constant throughout the optimization proced
 To do this, we need to define the variables `x` and `y`
 ```jl
 x = [1.0, 2.0, 3.0]
-y = 1.0 + 2.0 * x + [-0.3, 0.3, -0.1]
+y = 1.0 .+ 2.0 * x + [-0.3, 0.3, -0.1]
```
Comment (fredrikekre) [inline docs/src/user/tipsandtricks.md:15]: Perhaps dot it all the way?
```suggestion
y = 1.0 .+ 2.0 .* x .+ [-0.3, 0.3, -0.1]
```
or
```suggestion
y = @. 1.0 + 2.0 * x + [-0.3, 0.3, -0.1]
```

Comment (eirikbrandsaas) [reply docs/src/user/tipsandtricks.md:15]: That seems reasonable! I'll try to do this once I'm finished with a few pressing deadlines. PS: thank you for a great package and all the work you all do

Comment (fredrikekre) [reply docs/src/user/tipsandtricks.md:15]: I haven't actually contributed to this package, this was just a drive-by comment :)

Comment (pkofod) [reply docs/src/user/tipsandtricks.md:15]: I'll accept the complements then :) and thanks for fixing these issues

Comment (pkofod) [reply docs/src/user/tipsandtricks.md:15]: But let me thank @fredrikekre for giving these suggestions :)

**Comment by eirikbrandsaas** [comment]
Is there something else I should before you can merge this in?

**Comment by pkofod** [comment]
Yes, ping me ping me ping me :) Thanks

**Pull request merged**